data/json/items/comestibles/veggy_dishes.json

```diff
@@ -285,12 +286,13 @@
     "volume": 1,
     "flags": [ "EATEN_HOT" ],
     "fun": 3,
-    "vitamins": [ [ "vitC", 2 ], [ "calcium", 3 ], [ "iron", 2 ] ]
+    "vitamins": [ ["vitA", 1 ], [ "vitC", 9 ], [ "calcium", 13 ], [ "iron", 8 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "cooked_pumpkin",
     "name": "mashed pumpkin",
+    "weight": 259,
```

Comment:
this shouldn't be necessary, because of copy-from

Reply:
Removed, fixed.